backends/qualcomm/scripts/build.sh

```diff
@@ -71,7 +71,6 @@ if [ "$BUILD_AARCH64" = true ]; then
         -DCMAKE_INSTALL_PREFIX=$BUILD_ROOT \
         -DEXECUTORCH_BUILD_QNN=ON \
         -DEXECUTORCH_BUILD_SDK=ON \
-        -DFLATCC_TEST=OFF \
```

Comment:
Any specific reason we turn it on?

Reply:
No, we explicitly turn OFF it before.
Because recently [PR 2466](https://github.com/pytorch/executorch/pull/2466) turn it off, we don't need to set it again here.